internal/databroker/server.go

```diff
@@ -86,7 +86,7 @@ func (srv *Server) initVersion() {
 	}
 
 	// Get version from storage first.
-	if r := dbServerVersion.Get(context.Background(), serverVersionKey); r != nil {
+	if r, _ := dbServerVersion.Get(context.Background(), serverVersionKey); r != nil {
```

Comment:
Should we handle?

Reply:
If we can't get the version from storage, then we start with a fresh one, so I don't think what else we can do here.